gloo/transport/ibverbs/pair.cc

```diff
@@ -21,9 +22,11 @@ namespace transport {
 namespace ibverbs {
 
 Pair::Pair(
+    int rank,
```

Comment:
To make sure my understand correctly, this is always global rank right? For example, for Sub-PG scenario, we sometimes will have local ranks as well.

Reply:
This is the rank of the Gloo process group/context. On the gloo side there's only a single rank concept per context, no understanding of sub PG ranks etc.